packages/crud/src/vaadin-crud.js

```diff
@@ -19,8 +19,8 @@ import { defineCustomElement } from '@vaadin/component-base/src/define.js';
 import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
 import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { crudStyles } from './styles/vaadin-crud-core-styles.js';
```

Comment:
Should also update import in `vaadin-crud-dialog.js`

Reply:
I've decided to postpone extracting the core styles. I'll handle that in a separate step.